src/main/java/org/mskcc/smile/service/impl/ValidRequestCheckerImpl.java

```diff
@@ -214,7 +214,7 @@ public Boolean isValidNonCmoSample(Map<String, String> sampleMap)
 
     @Override
     public Boolean isCmo(String json) throws JsonProcessingException {
-        if (json == null || Strings.isBlank(json)) {
+        if (json == null || isBlank(json)) {
```

Comment:
Does String.isBlank check for null?  If so, we can remove json==null check.  If not, add it to isBlank().